.circleci/config.yml

```diff
@@ -0,0 +1,48 @@
+version: 2.1
+
+orbs:
+  ruby: circleci/ruby@1.1.0
+  node: circleci/node@2
+
+jobs:
+  build:
+    docker:
+      - image: cimg/ruby:3.1.0-node
+    steps:
+      - checkout
+      - ruby/install-deps
+      # Store bundle cache
+  test:
+    parallelism: 3
+    docker:
+      - image: cimg/ruby:3.1.0-node
+      - image: circleci/postgres:9.5-alpine
+        environment:
+          POSTGRES_USER: circleci-ruby
+          POSTGRES_DB: magma_clothing_estore_CarlosSe_test
+          POSTGRES_PASSWORD: ""
+    environment:
+      BUNDLE_JOBS: "3"
+      BUNDLE_RETRY: "3"
+      PGHOST: 127.0.0.1
+      PGUSER: circleci-ruby
+      PGPASSWORD: ""
+      RAILS_ENV: test
+    steps:
+      - checkout
+      - ruby/install-deps
+      - run:
+          name: Wait for DB
+          command: dockerize -wait tcp://localhost:5432 -timeout 1m
+      - run:
+          name: Database setup
+          command: bundle exec rails db:schema:load --trace
```

Comment:
you're missing the step that actually run all the specs, something like: `bundle exec rspec spec/`

Reply:
In this stage RSpec is not installed, I'll create the step in the CI after integrating and configuring Solidus  😄